src/js/plugins/inheritance.js

```diff
@@ -1634,6 +1634,35 @@
 
     init: function () {
       var self = this;
+      
+      // Watch the body for attribute changes indicating a card has loaded.
+      // This function uses a MutationObserver to check if the body has any
+      // attribute change (which loading a card will cause). If a card loads,
+      // we immediately call updateCardView which mounts the "Parent," 
+      // and "Child," buttons.
+      (function () {
+        // Target element that we will observe
+        const target = document.body;
```

Comment:
This seems like it would fire whenever anything in the DOM's body changed. Would it be possible to use `document.baseURI` instead to minimize unnecessary firings of the callback?

Reply:
Good idea, but in fact, we can't (I think). MutationObserver can only watch DOM changes and the baseURI is not a `Node` so it can't be "observed" with this API.

Reply:
There may in fact be a better `Node` to watch however, it seems like the MutationObserver only fires a few times since Trello boards are relatively static, there isn't much in the body that changes and we're only watching to see if an attribute changes.